IDEAS/Electric/Distribution/BaseClasses/Branch.mo

```diff
@@ -12,7 +12,7 @@ model Branch
   Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_a port_a annotation (
       Placement(transformation(extent={{10,10},{30,30}}), iconTransformation(
           extent={{10,10},{30,30}})));
-  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow prescribedHeatFlow
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow prescribedHeatFlow(alpha=1)
```

Comment:
@arnoutaertgeerts @Mathadon if we work conditionally, note that you only make the thermal connection conditional, and **not** the calculation of Plos, cause knowing the resistive losses are key in models.